changelog.d/clickhouse_acknowledgements.breaking.md

```diff
@@ -0,0 +1 @@
+Requests that encounter 500-level errors from the Clickhouse server will be retried instead of immediately producing an error at their source.
```

Comment:
```suggestion
When end-to-end acknowledgments are enabled, outgoing requests in the ClickHouse sink that encounter 500-level errors will now correctly report an errored (retriable) status, rather than a rejected (permanent) status, to Vector's clients.
```